components/atom/textarea/src/index.scss

```diff
@@ -4,7 +4,9 @@
 $short-num-lines: 5 !default;
 $long-num-lines: 7 !default;
 
-$pm-atom-textarea: $p-m !default;
+$p-atom-textarea: 2px !default;
```

Comment:
This is breaking the component API, we should try to keep previous compatibility while adding the new feature. Otherwise, we should publish a major and could be problematic to spread the new version. :)